event_sourcery/event_store.py

```diff
@@ -27,12 +26,20 @@ def __init__(
         self._outbox_storage_strategy = outbox_storage_strategy
         self._event_registry = event_registry
 
-    def outbox(self, publisher: Publisher) -> Outbox:
-        return Outbox(
-            storage_strategy=self._outbox_storage_strategy,
-            event_registry=self._event_registry,
-            publisher=publisher,
-        )
+    def run_outbox(
+            self,
```

Comment:
Double indent? :P Black should reformat this

Reply:
I prefer it this way, but I will change it. Sorry